Assets/Mirage/Runtime/ServerObjectManager.cs

```diff
@@ -416,6 +416,11 @@ internal void SpawnObject(GameObject obj, INetworkPlayer ownerPlayer)
                 throw new InvalidOperationException("SpawnObject " + obj + " has no NetworkIdentity. Please add a NetworkIdentity to " + obj);
             }
 
+            if (CheckForPrefab(obj))
+            {
+                throw new InvalidOperationException("GameObject " + obj.name + " is a prefab, it can't be spawned. This will cause errors in builds.");
```

Comment:
```suggestion
                throw new InvalidOperationException($"GameObject {obj.name} is a prefab, it can't be spawned. This will cause errors in builds.");
```

can we use string interpolation instead of concatenation, makes it easier to read